### DISCUSSION TITLE
timestamp unit

### DISCUSSION THREAD

**Discussion by giovanny07** [grafana/grafana#39602] [Dashboards, Panels & Visualizations]
Incorporate native timestamp reading to obtain epoch date like [https://www.epochconverter.com/](url)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
